force-app/main/default/classes/PerformanceScenarioRunner.cls

```diff
@@ -4,19 +4,28 @@
  */
 
 public inherited sharing class PerformanceScenarioRunner {
-    
     private PerformanceScenario performanceScenario;
+    private PerformanceTestingMode mode;
 
-    public PerformanceScenarioRunner(PerformanceScenario performanceScenario) {
+    public PerformanceScenarioRunner(PerformanceScenario performanceScenario, PerformanceTestingMode mode) {
```

Comment:
Would it make more sense to pass in the measure rather than the mode here? The runner doesn't really need to know that PEs don't support CPU time properly. It should just measure what it's told to measure.